developer-workflow/development-cycle.rst

```diff
@@ -34,7 +34,7 @@ and *null* for final releases), and ``N`` is the release serial number.
 Some examples of release tags: ``v3.7.0a1``, ``v3.6.3``, ``v2.7.14rc1``.
 
 Branches
-''''''''
+^^^^^^^^
```

Comment:
Should this use `-`? It seems to be the second level (a "subsection"), and according to https://devguide.python.org/documentation/markup/#sections, the second level should use `-`, while the _third_ level (such as lines 46 and 68 in this file) should use `^`

Reply:
Oh my, yeah you are right. This should be `---`, and we'll have to adjust all the following sub-stuff.

Reply:
Fixed in dc285783624e6abc26a7ee5391b37f5b03eccdd3